packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -35,8 +35,8 @@ function ListViewBlockSelectButton(
 	ref
 ) {
 	const blockInformation = useBlockDisplayInformation( clientId );
-	const { canMove, canRemove } = useBlockLock( clientId );
-	const isLocked = ! canMove || ! canRemove;
+	const { canEdit, canMove, canRemove } = useBlockLock( clientId );
+	const isLocked = ! canEdit || ! canMove || ! canRemove;
```

Comment:
`isLocked = ` keeps popping up – I'd find it convenient if that was returned by the `useBlockLock`

Reply:
100%. It's easier to derive values initially and then make things part of the hook once you see the pattern.

Reply:
Done - [71ebda8](https://github.com/WordPress/gutenberg/pull/39950/commits/71ebda894c35bcc17521fb8ad30b68bc5927c332)